Resources/Prototypes/tags.yml

```diff
@@ -689,6 +689,9 @@
 - type: Tag
   id: MagazineShotgun
 
+- type: Tag
+  id: MagazineUniversalMagnum
```

Comment:
Put the tag in Resources/Prototypes/DeltaV/tags.yml or Resources/Prototypes/Nyanotrasen/tags.yml
No need to change basegame files